### DISCUSSION TITLE
Pepe

### DISCUSSION THREAD

**Discussion by leyou4** [community/community#137775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Building flutter tool...
Running pub upgrade...
Resolving dependencies...
Got dependencies.
Downloading Material fonts...
Downloading Gradle Wrapper...
Downloading package sky_engine...
Downloading flutter_patched_sdk tools...
Downloading flutter_patched_sdk_product tools...
Downloading windows-x64 tools...
Downloading windows-x64/font-subset tools...

**Comment by dhruv-ambaliya** [comment]
Hi @leyou4 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question or feedback entails, giving a few more details might help someone give you a nudge in the right direction.
